baremetal.go

```diff
@@ -0,0 +1,114 @@
+// Package vagrantssh provides host connectivity in go for system/integration
+// testing in a multi host environment. It supports two testbed environments viz.
+// baremetal and vagrant
+/*
```

Comment:
Yeah right here: https://github.com/contiv/vagrantssh/blob/master/vagrant.go#L1

Can you merge your additions into that?